ai-missions/high-risk-flu-shot-mission/skills/send_virtual_care_notification/Readme.md

```diff
@@ -0,0 +1,35 @@
+### Send Virtualcare Notification
+
+
+#### Description:
+Shares given profile details with Virtual care nurse to call and inform regarding flu-shots.
```

Comment:
For the Profile IDs derived from the configured Cohort, member contact information is shared with virtual care nurses, who call members to provide information and schedule flu-shots.